test/lib/runners/webpackRunner.js

```diff
@@ -0,0 +1,90 @@
+const BaseRunner = require('./baseRunner'),
+  path = require('path'),
+  // webpack and html-webpack-plugin are globally installed in the container
+  webpack = require('/usr/local/lib/node_modules/webpack'),
+  HtmlWebpackPlugin = require('/usr/local/lib/node_modules/html-webpack-plugin');
+
+async function buildWithWebpack(snippet) {
+  const config = {
+    entry: snippet.renderedSnippetPath,
+    output: {
+      path: path.resolve(
+        __dirname,
+        '..',
+        '..',
+        'bin',
+        'sdk-js',
+        'webpack',
+        snippet.name
+      ),
+      filename: 'build.js'
+    },
+    module: {
+      rules: [
+        {
+          test: /\.js$/,
+          loader: 'babel-loader'
+        }
+      ]
+    },
+    devtool: 'eval-source-map',
+    plugins: [new HtmlWebpackPlugin()]
+  };
+
+  return new Promise((resolve, reject) => {
+    webpack(config, (err, stats) => {
+      if (err) {
+        if (err.details) {
+          reject(err.details);
```

Comment:
You should break the control flow:
```suggestion
          return reject(err.details);
```